src/AccountingModule.sol

```diff
@@ -173,7 +242,7 @@ contract AccountingModule is IAccountingModule, Initializable, AccessControlUpgr
      * @param _lowerBound in bips, as a function of % of tvl
      * @dev hard max of 50% of tvl
      */
-    function setLowerBound(uint16 _lowerBound) external onlyRole(SAFE_MANAGER_ROLE) {
+    function setLowerBound(uint256 _lowerBound) external onlyRole(SAFE_MANAGER_ROLE) {
         if (_lowerBound > (DIVISOR / 2)) revert InvariantViolation();
```

Comment:
create a constant `uint256 constant MIN_LOWER_BOUND = DIVISOR / 2` and use that instead. No need to divide by 2 everytime this function is called, plus makes the code more readable.